utils/src/utils/openUrl.ts

```diff
@@ -3,11 +3,9 @@
  *  Licensed under the MIT License. See License.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import open = require("open");
+import * as vscode from 'vscode';
 
 export async function openUrl(url: string): Promise<void> {
     // Using this functionality is blocked by https://github.com/Microsoft/vscode/issues/85930
-    // await vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(url));
-
-    await open(url);
+    await vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(url));
```

Comment:
Might need to make changes in the default webpack config in the dev package too, with `open` removed